pkg/application/inject/fuse/injector.go

```diff
@@ -164,12 +164,16 @@ func (s *Injector) inject(in runtime.Object, runtimeInfos map[string]base.Runtim
 			continue
 		}
 
+		if err = s.injectCheckMountReadyScript(pod, runtimeInfos, fake.NullLogger()); err != nil {
```

Comment:
I think fake package should be used in go test  file.

Reply:
Thx. Fixed